collect.py

```diff
@@ -1,16 +1,17 @@
 from concurrent.futures import ThreadPoolExecutor
+import glob
 import json
 import os
+import random
 import sys
 import time
-import random
 import traceback
-import glob
+
 from PIL import Image
-from selenium import webdriver
-from selenium.common.exceptions import NoAlertPresentException, NoSuchWindowException, TimeoutException
 
 from autowebcompat import utils
+from selenium import webdriver
```

Comment:
The selenium imports are third-party libraries, so they should go in the second section of the imports. This is probably the cause of at least one of the flake8 failures.